Client/Frontend/Widgets/PhotonActionSheet.swift

```diff
@@ -198,8 +200,9 @@ class PhotonActionSheet: UIViewController, UITableViewDelegate, UITableViewDataS
         super.viewDidLayoutSubviews()
         let maxHeight = self.view.frame.height - (self.showCloseButton ? PhotonActionSheetUX.CloseButtonHeight : 0)
         tableView.snp.makeConstraints { make in
+            heightConstraint?.deactivate()
             // The height of the menu should be no more than 80 percent of the screen
-            make.height.equalTo(min(self.tableView.contentSize.height, maxHeight * 0.8))
+            heightConstraint = make.height.equalTo(min(self.tableView.contentSize.height, maxHeight * 0.8)).constraint
         }
         if style == .bottom && self.modalPresentationStyle == .popover {
```

Comment:
This check is in a few places in the code, can this be a getter that explains why a bottom popover has particular conditions over other combinations.

Reply:
The new popover case in the enum should fix this.